rct229/ruletest_engine/ruletest_jsons/scripts/excel_to_test_json_utilities.py

```diff
@@ -980,15 +980,15 @@ def add_plant_loop_equipment(template_rmd, system_rmd):
         "heat_rejections",
     ]
 
-    sys_ruleset_model_instance = system_rmd["ruleset_model_instances"][0]
+    sys_ruleset_model_instance = system_rmd["ruleset_model_descriptions"][0]
```

Comment:
Would it be more clear if we changed the ```sys_ruleset_model_instance``` variable name to ```sys_ruleset_model_descriptions```?

Reply:
Yes it would! Thanks for catching that.